frontend/components/SetupPage/Create/SetupPassword.tsx

```diff
@@ -14,16 +16,42 @@ import { SetupCreateHeader } from './SetupCreateHeader';
 
 const { Title, Text } = Typography;
 
+const PasswordStrength = ({ score }: { score: number }) => {
+  const strength = [
+    'Too weak',
+    'Weak',
+    'Moderate',
+    'Strong',
+    'Very strong! Nice job!',
+  ];
+  const colors = [
+    COLOR.RED,
+    COLOR.WARNING,
+    COLOR.SUCCESS,
+    COLOR.SUCCESS,
+    COLOR.PURPLE,
+  ];
```

Comment:
let's place them outside the component, otherwise the'll be recreated on every render when the score changes